dev-image-manifest.aib.yml

```diff
@@ -0,0 +1,140 @@
+name: dev-image
+
+content: 
+  repos:
+    - id: copr-sample-apps
+      baseurl: https://download.copr.fedorainfracloud.org/results/alexl/cs9-sample-images/centos-stream-9-$arch/
+  rpms:
+    - auto-apps
+    - vsomeip3
+    - vsomeip3-devel
+    - vsomeip3-routingmanager
+    - dlt-daemon
+    - boost-system
+    - boost-thread
+    - boost-log
+    - boost-chrono
+    - boost-date-time
+    - boost-atomic
+    - boost-filesystem
+    - boost-regex
+    - boost-devel
+    - vim
+    - git
+    - make
+    - cmake
+    - gdb
+    - gcc-c++
+    - openssh-server
+    - iproute
+    - dnf
+    - tar
+    - osbuild
+    - python3.12
+    - python3.12-pip
+    - wget
+    - podman
+    - automotive-image-builder
+
+  add_files:
+    - path: /oc-arm64-rhel9.tar.gz
+      url: https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-4.18/openshift-client-linux-arm64-rhel9.tar.gz
+    
+    - path: /tkn.rpm
+      url: https://github.com/tektoncd/cli/releases/download/v0.40.0/tektoncd-cli-0.40.0_Linux-ARM64.rpm
+
+    - path: /usr/local/bin/caib
+      url: https://github.com/rh-sdv-cloud-incubator/automotive-dev-operator/releases/download/v0.0.7/caib-v0.0.7-arm64
+
+    - path: /usr/lib/systemd/system/tkn-install.service
+      text: |
+        [Unit]
+        Description=Extract Tekton CLI tarball
+        After=network-online.target
+        ConditionPathExists=/tkn.rpm
+
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/dnf install -y /tkn.rpm
+        RemainAfterExit=true
+
+        [Install]
+        WantedBy=multi-user.target
+
+    - path: /usr/lib/systemd/system/oc-install.service
+      text: |
+        [Unit]
+        Description=Download and extract OpenShift CLI (oc + kubectl)
+        After=network-online.target
+        ConditionPathExists=/oc-arm64-rhel9.tar.gz
+
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/tar -xzf /oc-arm64-rhel9.tar.gz -C /usr/local/bin
+        ExecStartPost=/usr/bin/chmod +x /usr/local/bin/oc /usr/local/bin/kubectl
+        RemainAfterExit=true
+
+        [Install]
+        WantedBy=multi-user.target
+s
+    - path: /usr/lib/systemd/system/caib-install.service
```

Comment:
do we need this?

Reply:
yes, we need to add the +x permission, or else it's not executable.

Reply:
I believe you can use chmod_files for this